daemon/src/taker_cfd.rs

```diff
@@ -60,6 +60,21 @@ enum RollOverState {
     None,
 }
 
+pub struct RollOverParams {
+    settlement_interval: Duration,
```

Comment:
```suggestion
    /// Defines the maximum duration a CFD may be open for to still qualify for automatic roll-over.
    max_open: Duration,
```

cc @da-kami @bonomat

Reply:
I would prefer `settlement_interval` or something with `*interval` in it.

Reply:
Why interval though? An interval is a duration that happens regularly. This value is not an interval. If you look at the logic how this is used, it only represents a cut-off time after which we no longer trigger roll over.

Reply:
More concrete on `max_open` - I think in the context of rollover this name does make more sense than `settlement_interval`. 

---

A few words about different name on the "same" thing in a different context. 

Similar to the oracle's `announcement_lookahead` this variable has to be aligned with the `settlement_interval`. 
Initially we had a `term` that was pulled through the codebase and it was clear what it was, because it was named the same everywhere. That caused confusion in the context of a specific actor, but was easy to understand in "the big picture". Now I feel we have the opposite - in the context of each actor we try to find a name, but the big picture is hard to grasp.

I am not sure we can call it "an implementation detail" that the cfd's `settlement_interval` has to be used for the oracle's `announcement_lookahead` and the rollover's `max_open` - if those values are not aligned weird side effects can happen because we use different time constraints that are supposed to be aligned.

That said, maybe we should take a step back and think about creating a component that encapsulates this knowledge. I don't want to block this PR on this, but this is not the first time we run into this issue.

Reply:
I don't think it has to be aligned, has it? `max_open` should be **equal or less** than `settlement_interval`. 

> That caused confusion in the context of a specific actor,

I don't think I was part of this conversation. Can you provide more context?

If a value MUST be the same across multiple parts of the system for it to work then we should absolutely have it as a single value.

Reply:
> I don't think it has to be aligned, has it? max_open should be equal or less than settlement_interval.

It has to be equal to make sense. It *can* be less for the system to still work, but that is out of the context of what we want to achieve. 

> I don't think I was part of this conversation. Can you provide more context?

Yes, this happened when you were on vacation 😬
Here is the PR:
https://github.com/itchysats/itchysats/pull/424

> If a value MUST be the same across multiple parts of the system for it to work then we should absolutely have it as a single value.

I am not sure the rename was a good idea. The rename is correct in the context of each actor, but the fact that the same value has to be used gets lost.

@rishflab and me are adding a commit that uses exactly *one* constant to represent the value (i.e. the 24 hours).
I hope that helps.